apisix/plugins/ocsp-stapling.lua

```diff
@@ -46,25 +46,34 @@ function _M.check_schema(conf)
 end
 
 
-local function fetch_ocsp_resp(der_cert_chain)
-    core.log.info("fetch ocsp response from remote")
-    local ocsp_url, err = ngx_ocsp.get_ocsp_responder_from_der_chain(der_cert_chain)
-
-    if not ocsp_url then
+local function get_ocsp_responder(der_cert_chain)
```

Comment:
can we skip this unnecessary function name modification/breaking down of one function into two?

I am not a OCSP expert and it makes difficult to review.

Reply:
break down this function because we could reuse it in client verify.